blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/AbstractRunImpl.java

```diff
@@ -83,7 +82,7 @@ public BlueRunState getStateObj() {
         } else if(!run.isLogUpdated()){
             return BlueRunState.FINISHED;
         } else {
-            return BlueRunState.QUEUED;
```

Comment:
Agreed - I need to look into this. The non-pipeline version of this should never return queued as it will receive a `BlueQueuedRun` instead of this model